apps/startrek/src/styles/globals.css

```diff
@@ -0,0 +1,17 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  html {
+    font-family: Lexend, sans-serif;
```

Comment:
Are we using this font or it is a placeholder?

Reply:
This font was not referenced from our UI, it's just a placeholder font to check if the configuration is correct in tailwindcss